packages/babel-traverse/src/path/introspection.js

```diff
@@ -160,12 +160,15 @@ export function isStatementOrBlock() {
 export function referencesImport(moduleSource, importName) {
   if (!this.isReferencedIdentifier()) {
     if (
-      this.isMemberExpression() &&
-      !this.node.computed &&
-      this.node.property.name === importName
+      (this.isMemberExpression() || this.isOptionalMemberExpression()) &&
+      ((!this.node.computed && this.node.property.name === importName) ||
+        (this.node.computed &&
+          t.isStringLiteral(this.node.property) &&
+          this.node.property.value === importName))
```

Comment:
Nit: I would find this code easier to understand, since it makes the division between the computed/not computed cases clearer:

```suggestion
      (this.node.computed
        ? t.isStringLiteral(this.node.property, { value: importName })
        : this.node.property.name === importName)
```

Reply:
I don't really have a preference so I'll accept the suggestion :sweat_smile: thanks, hope you got the Prettier format right :P